docs/content/5.integrations/2.guide/2.adding-headers-to-request.md

```diff
@@ -0,0 +1,284 @@
+# Pass Custom Headers in Requests
+
+In this tutorial, we'll demonstrate how to set up an interceptor to pass headers when making requests using the axios client inside the Vue Storefront SDK. This is useful when you need to append custom headers or cookies to your API calls.
+
+After passing the headers to the axios client, we'll also show you how to pass them to external services using the API Client.
+
+::danger
+Do not use createSharedComposable if creating a shared composable that uses the interceptor. This will cause the interceptor to only be created once, which will result in the headers being shared across instances.
```

Comment:
It's a bit out of context here. A newcomer might have no idea what `createSharedComposable` is and why would he/she even has to use it. Perhaps this danger fits elsewhere?

Reply:
I agree it's no longer necessary. We have since removed this composable from our projects.